_collections/_news/2024-12-14-2024-25s-semester-1-overview.md

```diff
@@ -0,0 +1,151 @@
+Lua mug and a laptop with Lua on it.](/assets/images/2024-semester-1/lua-workshop.png)
+
+After that, we leaned back into our social side and ran our annual Halloween Quiz, where we saw a vast range of bizarre
+costumes and teams battling for wit to win a swarm of Blahajs, and an Aftonsparv for the costume competition! Despite
+a marking error at the end that did end up affecting the final scores (oops), the event went wonderfully and it was 
+incredible to see so many people dressed up.
+
+![A picture of the Halloween Quiz inside of the Computer Science atrium.](/assets/images/2024-semester-1/halloween-quiz.png)
+
+Then, sometime in between, we got declared war upon by Bad Film Society, where they egregiously accused us of using their
+"brand identity" for some harmless publicity... needless to say, I will break my silence on this matter and will confirm
+that I have not tolerated this baseless slander, and that we have spoken diplomatically to find a negotiation that will 
+appease both parties, but I fear that we will have to resort to measures that will take us both down, as I will not 
+kneel to Bad Film's unreasonable demands and let them step over CSS like a doormat. We will win this war, one way or 
+another!! 🔥🔥🔥
+
+(For anyone struggling to read the post, you can see it in full here: <https://www.instagram.com/p/DCgljIPtNO2/> )
+
+![A screenshot of Bad Film's declaration of war.](/assets/images/2024-semester-1/bad-film-war.png)
+
+Towards the end of the semester, we rounded off with three big events - this being the Bar Crawl, our usual Christmas Meal
+and the return of the legendary Discord Botathon!
+
+On the 30th of November, we hosted a bar crawl that traversed throughout the city of Birmingham, starting at Mr Mulligan's,
+making our way to the energetic Heidi's, partying our way to the Brasshouse before finally finishing off at the Soloman
+Cutler, concluding an incredibly successful bar crawl.
+
+![A picture of members at the bar crawl inside the Soloman Cutler.](/assets/images/2024-semester-1/bar-crawl.png)
+
+Following that, on the 5th of December, we held our Christmas Meal, returning once again to the Green Man in Harborne!
+We just happened to visit on a quiz night as well, so some groups of CSS members formed teams and participated in the
+quiz - one getting second place, and another getting second to last. Regardless, it was a wonderful way to conclude the
+semester, and it was wonderful to see so many people there!
+
+![A picture of members at the Christmas Meal in the Green Man, some students sitting at tables.](/assets/images/2024-semester-1/christmas-meal.png)
+
+And lastly, on the 8th of December, we were able to host the Discord Botathon - the last iteration being held in 2021.
+Seven different teams responded to the call and built a Discord bot from scratch, learning from the bot workshop held 
+the day beforehand. Here's how the final standings looked:
```

Comment:
```suggestion
Finally, on the 8th of December we hosted the Discord Botathon, the first since 2021.
Seven different teams responded to the call and built a Discord bot from scratch, some learning from the bot workshop held 
the day before. Here's how the final standings looked:
```